### DISCUSSION TITLE
He

### DISCUSSION THREAD

**Discussion by HbmMods** [HbmMods/Hbm-s-Nuclear-Tech-GIT#503] [Announcements]
![2022-03-12_16 58 20](https://user-images.githubusercontent.com/16674258/158025237-8d80d490-e4c7-4f95-a491-c075ee62e73b.png)

**Comment by Nos74** [comment]
He

**Comment by ncr-t** [comment]
IT SPINS AHAHHAAHAHAHAHAHHA

**Reply by HbmMods** [reply]
Not only does it spin, but it slowly accelerates when starting.

**Reply by ncr-t** [reply]
zamn ?!!?!?
(don’t kill me bob)

**Reply by HbmMods** [reply]
huh

**Comment by SoB1nary** [comment]
he.

**Comment by 7pheonix** [comment]
yup, that's he alright

**Comment by HbmMods** [comment]
![image](https://user-images.githubusercontent.com/16674258/158026437-ad64573b-9b54-4fb6-b8e0-d88b6729aedf.png)

All concrete

**Reply by ncr-t** [reply]
(woops !)

**Reply by SoB1nary** [reply]
das conc creet baybee
(thank you very much for a way to concrete faster. i will use this power responsibly)

**Reply by HbmMods** [reply]
![image](https://user-images.githubusercontent.com/16674258/158026498-25080772-5ae0-4aeb-a9dd-ac9c3f063971.png)
it's been like two minutes, and I haven't even installed upgrades yet

**Comment by HbmMods** [comment]
![image](https://user-images.githubusercontent.com/16674258/158026639-c786821a-07ac-483a-81fa-3b36e1d557e6.png)

**Reply by ncr-t** [reply]
speed
